### ISSUE TITLE
Discovery of account management

### ISSUE THREAD

**Issue by megoth** [solid/data-interoperability-panel#18]
The various implementations might differ in how they allow users to manage their accounts. Should the data that describes these options be standardized somehow, so that various clients can present them to the users?

Example: Alice wants to delete her account on `https://alice.solid.community`. How does she know that to delete her account she must visit `https://alice.solid.community/account/delete` ?

Does she have to go through the account provider's UI, should there be standardized APIs, or at least a data model that describes the various possibilities for account management?

**Comment by megoth** [comment]
If this is outside the scope of this Panel, let me know.

**Comment by RubenVerborgh** [comment]
> Should the data that describes these options be standardized somehow, so that various clients can present them to the users?

Yes, IMHO. Was also @timbl's suggestion.

Even created a spec document for that already: https://github.com/solid/specification/blob/master/pod-management/index.bs

> should there be standardized APIs, or at least a data model

Yes.

As far as I'm concerned:
- the LDP API
- with specific data models

This would be a case where I would say that we agree on _exact_ ontologies and spec them. (As opposed to data guidance for data that is inside of the pod.)

> If this is outside the scope of this Panel, let me know.

Definitely in scope, IMHO.

**Comment by RubenVerborgh** [comment]
Can apply to both. Storage accounts are accounts too, with or without identity.

**Comment by RubenVerborgh** [comment]
> * Important to disambiguate Identity Provider and Resource Server.

The same API might be used for both though.
(e.g., `providesWebId` might just be a predicate)

> * Operations on the Resource Server (data pod)

Here, we will need to be careful with terminology.
It's not the data pod, but the level above it.

For simplicity, we could just call them "account management operations".

> Parental control over account

I would leave this out of the initial scope, this is much more complex than the rest.

**Comment by kjetilk** [comment]
Another disambiguation I think we need to do is between how the Pod Provider manages accounts on their system, and how the user manages their own account.

Perhaps we should clearly scope this issue to the latter? Supposedly, the former will use methods from the former to do work, but it is a lot more work, and also difficult policy issues associated with it.

**Comment by kjetilk** [comment]
> > Parental control over account
> 
> I would leave this out of the initial scope, this is much more complex than the rest.

Yeah, the full scope of Parental Control is really hard. Initially, it could be thought of us just about setting some `acl:Control`s when creating a Pod, and so, a way to write the ACLs that initializes the Pod could suffice, but it is more a Pod Provider problem than a user account problem. So, as per the above, perhaps we should set the scope of this project to be user account only, and Parental Control would be a part of a future Pod Provider management project?
